recipes-devtools/mraa/mraa.inc

```diff
@@ -9,7 +9,7 @@ S = "${WORKDIR}/git"
 
 inherit distutils-base pkgconfig python-dir cmake
 
-EXTRA_OECMAKE_append = "-DINSTALLTOOLS=ON -DFIRMATA=ON -DPYTHON_SITE_DIR:FILEPATH=${PYTHON_SITEPACKAGES_DIR} -DBASE_LIB_INSTALL_DIR=${base_libdir} -DCMAKE_SKIP_RPATH=ON"
+EXTRA_OECMAKE_append = "-DINSTALLTOOLS=ON -DFIRMATA=ON -DPYTHON2_PACKAGES_PATH:FILEPATH=../${PYTHON_SITEPACKAGES_DIR} -DBASE_LIB_INSTALL_DIR=${base_libdir} -DCMAKE_SKIP_RPATH=ON"
```

Comment:
The problem is that PYTHON_SITEPACKAGES_DIR includes /usr whilst PYTHONx_PACKAGES_PATH expects it without the prefix so it ends up being /usr/usr. since python-dir class doesn't expose any other vars I can use I figured that was the "best" fix.